metricbeat/module/system/core/core.go

```diff
@@ -45,10 +56,47 @@ func (m *MetricSet) Fetch() ([]common.MapStr, error) {
 	cores := []common.MapStr{}
 
 	for core, stat := range cpuCoreStat {
-		coreStat := m.cpu.GetCpuStatEvent(&stat)
```

Comment:
Why didn't you do the changes in the `GetCpuStatEvent` function? So, `system` type doc can benefit from it.